cypress/config/cypress.config.ts

```diff
@@ -14,13 +14,13 @@ export default defineConfig({
     specPattern: './**/e2e/**/*.cy.{js,jsx,ts,tsx}',
     supportFile: './cypress/support/e2e.ts',
     fixturesFolder: './cypress/fixtures',
-    defaultCommandTimeout: 10000,
+    defaultCommandTimeout: 10_000,
```

Comment:
please revert. It's some strange idea from bot

Reply:
Fixed